### DISCUSSION TITLE
PayPal Link Issue

### DISCUSSION THREAD

**Discussion by Hiro-Tokyo** [code4recovery/12-step-meeting-list#1248] [Q&A] [answered]
Hello Team!

PayPal Link does not work at the moment.
When we click PayPal button, it includes "%20" so any solution?

https://aajapan-online-intergroup.org/meetings/?weekday=saturday&meeting=2022-03-0079

https://**%20**www.paypal.com/paypalme/tokyoyoungpeople

**Comment by joshreisner** [comment, marked as answer]
thanks @Hiro-Tokyo for the report - should be fixed now 👍

**Comment by Hiro-Tokyo** [comment]
Thanks Josh! Yes, it's working now!
